src/status_im/ui/screens/keycard/onboarding/views.cljs

```diff
@@ -16,12 +16,15 @@
   (:require-macros [status-im.utils.views :refer [defview letsubs]]))
 
 (defview intro []
-  (letsubs [flow [:keycard-flow]]
+  (letsubs [flow [:keycard-flow]
+            {:keys [from-key-storage-and-migration?]} [:keycard]]
     [react/view styles/container
-     [topbar/topbar]
+     (when-not from-key-storage-and-migration?
+       [topbar/topbar])
```

Comment:
i feel like from the design we still need topbar space but without navigation

Reply:
I have added space to the container. See next few lines (look out for `margin-top`).